styling/src/jsMain/kotlin/dev/fritz2/styling/theme/default.kt

```diff
@@ -2126,6 +2129,10 @@ open class DefaultTheme : Theme {
             minWidth { "22vw" }
```

Comment:
@jwstegemann can you reply to this!? Thx.

Reply:
Proposal:
- use `15vw` as default (rely on KS demo to make that look nice)
- provide some extra parameter to `AppFrame` context to make this configurable, as this may be important for different apps within organisations to adapt individually! (Some menu entries might be wider than others; deriving and maintaining an extra theme just for that purpose is an overkill and cumbersome!)

Reply:
Sounds good in my opinion!

Reply:
Done :-)